docs/typelibrary/io/ID.md

```
@@ -1,3 +1,62 @@
-### ID - Input DWORD (32 Bit)
+# ID
 
 ![](https://user-images.githubusercontent.com/69573151/210779975-495c4210-22c0-4962-af45-e98d38b26edb.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)